### DISCUSSION TITLE
Instalar o Phpmyadmin e  Scada-LTS (Manual) No Linux -Ubunbu  e Windows

### DISCUSSION THREAD

**Discussion by lclaros85** [SCADA-LTS/Scada-LTS#3057] [General]
Good morning!

Could someone help me with performing the installations manually, step by step, on both systems? I tried using the script posted here, but the database manager required for my company is phpMyAdmin because it is more common. Anyway, SCADA and MySQL installed fine with the script, everything is working correctly, but it's not possible to manage it via PHP. I need help for both Linux and Windows. Could anyone assist me? Thank you in advance.

**Comment by Limraj** [comment]
Hi @lclaros85,
For both systems you have the option not to install the database locally.
You just have to visit the websites:
https://github.com/SCADA-LTS/linux-installer
https://github.com/SCADA-LTS/windows-installer

1. On Linux you don't invoke the mysql_start.sh script while when calling the script tomcat_start.sh at tomcat configuration, you enter the data of the MySQL database that you already have available;

2. And on Windows, in the installer, uncheck the "Install local server" checkbox. When configuring the MySQL server, you only enter the data of the existing running MySQL server;

This means that two installation scenarios are supported: when you already have the MySQL server installed and available, and when you need to install everything.

**Unfortunately, we have limited resources and cannot deal with everything at once. However, we encourage commercial cooperation. You would be in direct contact with us and your problems would be a priority.**

Regards.

**Reply by lclaros85** [reply]
tank you @Limraj  , We decided to migrate to Windows and it worked! I'll open a topic for another problem I have

**Reply by fabiodurao** [reply]
In english please @lclaros85

**Reply by lclaros85** [reply]
I'm sorry, I forgot @fabiodurao
